lib/search/index.ts

```diff
@@ -234,11 +235,11 @@ export const middleware: S.Middleware = (store) => {
       }
 
       const searchText = titleOnly ? getTitle(note.content) : note.content;
-
+      const normalizedSearchText = removeAccents(searchText);
       if (
         searchTerms.length > 0 &&
         !searchTerms.every((term) =>
-          searchText.includes(term.toLocaleLowerCase())
+          normalizedSearchText.includes(term.toLocaleLowerCase())
```

Comment:
the `content` in this file is already processed for searching and already does things like normalize and lower-case the text. there's no need to continue to recalculate it for every note on every search when we could add the transformation to the place where the search index is updated in response to note edits.